02_Spatial_data_and_R_packages_for_mapping.Rmd

```diff
@@ -0,0 +1,356 @@
+# Spatial data and R packages for mapping
+
+**Learning objectives:**
+
+- Spatial data types
+- CRS
+- making a map with R
+
+## Spatial data and R packages for mapping
+
+
+Type of Spatial data:
+
+- areal
+- geostatistical
+- point patterns
+
+```{r echo=FALSE, message=FALSE, warning=FALSE, paged.print=FALSE}
+library(tidyverse)
+```
+
+Packages used:
+
+- ggplot2
+- leaflet
+- mapview
+- tmap
+
+### Types of spatial data
+
+
+Define the space:
+
+$$\left\{Z(s):s\in D \subset \mathbf{R^d} \right\}$$
+
+$$Z=\text{observed values}$$
+$$s=\text{location}$$
+$$D=\text{domain (aereal, geostatistical, point patterns)}$$
+
+$$d=\text{dimension}$$
+
+
+#### Areal data
+
+The domain $D$ is **fixed** (of regular or irregular shape) and made of finite number of areal units with well-defined boundaries:
+
+- zip code
+- census tract
+- pixels
+
+```{r echo=TRUE, message=FALSE, warning=FALSE, paged.print=FALSE}
+require(sf)
```

Comment:
Move these "requires" up to the top block as `library()` calls. In general you should probably never use "require".